### PR TITLE
Add transactions resource and time field option

### PR DESCRIPTION
**Pull request by nathanielmiller20**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -300,6 +301,38 @@ puts response
 # => Button::Response()
 ```
 
+### Transactions
+
+#### All
+
+_n.b. all is a paged endpoint.  Take care to inspect `response.next_cursor` in case there's more data to be read._
+
+Along with the required account id, you may also pass the following optional arguments as a Hash as the second argument:
```
Comment (griffinmyers) [inline README.md]: Please update this line (no account id)

File: lib/button/resources/transactions.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
+require 'button/resources/resource'
+
+module Button
+  # https://www.usebutton.com/developers/api-reference/
+  #
+  class Transactions < Resource
+    # Gets a list of transactions
+    #
+    # @param [String] account_id the account id to look up transactions for
```
Comment (griffinmyers) [inline lib/button/resources/transactions.rb]: Please update docstring

File: lib/button/client.rb [hunk 3]
```diff
@@ -34,6 +35,7 @@ def initialize(api_key, config = {})
       @merchants = Merchants.new(api_key, config_with_defaults)
       @offers = Offers.new(api_key, config_with_defaults)
       @orders = Orders.new(api_key, config_with_defaults)
+      @tranasactions = Transactions.new(api_key, config_with_defaults)
```
Comment (griffinmyers) [inline lib/button/client.rb]: Looks like a misspelling, can you double check?

**Review by griffinmyers — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (nathanielmiller20) [reply README.md]: Fixed

File: lib/button/resources/transactions.rb (hunk 2 above)
Comment (nathanielmiller20) [reply lib/button/resources/transactions.rb]: Removed

File: lib/button/client.rb (hunk 3 above)
Comment (nathanielmiller20) [reply lib/button/client.rb]: Indeed! Thanks for the catch!

**Review by griffinmyers — APPROVED** [review]

**Pull request merged**